examples/tests/dependency_vmimage.py

```diff
@@ -0,0 +1,109 @@
+from avocado import Test
+from avocado.utils import vmimage
+
+
+class FedoraS390xVmimageTest(Test):
```

Comment:
I am not sure if it is needed to have examples for all supported images. IMO, it will be more work to maintain such example in times when we drop support of some versions. Also, IMO, having just one example is enough for users to understand how the vmimage runner works.